components/DEPS

```diff
@@ -52,8 +56,15 @@ specific_include_rules = {
   "webcompat_exceptions_unittest\.cc": [
     "+chrome",
     "+content",
+    "!brave/components/constants/brave_paths.h",
```

Comment:
why do we need this? isn't the "-" rule in `include_rules` sufficient?

Reply:
this allows it, but also shows up in `--count-violations`